components/back-top/back-top.component.ts

```diff
@@ -42,6 +43,8 @@ const passiveEventListenerOptions = normalizePassiveListenerOptions({ passive: t
   selector: 'nz-back-top',
   exportAs: 'nzBackTop',
   animations: [fadeMotion],
+  standalone: true,
+  imports: [BidiModule, NgIf, NgTemplateOutlet, NzIconModule, PlatformModule],
```

Comment:
suggestion: you can remove the PlatformModule because it export nothing and you can also remove the BidiModule because the directive exported is not used.